src/sections/CTASection/CTASection.tsx

```diff
@@ -50,7 +52,7 @@ export const CTASection = ({
     <HighlightPanel>
       <Container>
         <ContentWrapper>
-          <h3>{heading}</h3>
+          <h2 style={{ margin: '0px', fontWeight: 600 }}>{heading}</h2>
```

Comment:
Why do we pass inline styles? Cannot we define it analogically to what we had for `h3` in `HighlightPanel`?

Reply:
We can, but I thought that I will be moving all styles to the `custom.css` anyway while removing styled components so I decided to leave it like that. But it will be easier to move when we have everything styled the same way so I will fix it here :D